0000-rep-template.md

```diff
@@ -0,0 +1,45 @@
+---
+REP Number: <Fill me in with a four-digit unique incrementing number, matching the pull request number; update AFTER REP acceptance.>
+Author: <Author's First Name and Last Name.>
+Start Date: <Today's Date, YYYY-MM-DD>
+Bicep Issue Number: <Fill if the REP originates from a Bicep issue; otherwise, delete.>
```

Comment:
Could be multiple issues in some cases. Maybe we just need to add " (s)" to the title?

Reply:
Fixed.